deployment-guides/AWS-EKS-EFS/deploy_efs_shared_fs.sh

```diff
@@ -2,18 +2,18 @@
 
 echo "1. Creating EFS file system"
 
-aws efs create-file-system --creation-token "$FILESYSTEM_NAME" --performance-mode generalPurpose --throughput-mode bursting --region $AWS_REGION --tags Key=Name,Value="Galaxy filesystem" Key=developer,Value="$DEVELOPER"
+aws efs create-file-system --creation-token "$FILESYSTEM_NAME" --performance-mode maxIO --throughput-mode bursting --region $AWS_REGION --tags Key=Name,Value="Galaxy filesystem" Key=developer,Value="$DEVELOPER"
```

Comment:
Maybe here we could explain the differences and when you would want to use maxIO or generalPurpose?

Reply:
Using "General purpose" was the cause fo the original issues we got when using GCP NFS storage, hence why we moved to a higher performant system. EFS only offer maxIO as the better alternative. Using maxIO has never produced any errors and it does not cost the earth even for personal use, you only pay for what you are using. An organised galaxy setup should be very cheap to run.